product_docs/docs/biganimal/release/pricing_and_billing/index.mdx

```diff
@@ -26,6 +26,8 @@ If you're connecting directly to BigAnimal through BigAnimal's cloud account, ED
 
 If you're connecting directly to BigAnimal through your Microsoft Azure or AWS account, EDB doesn't bill you for cloud infrastructure, such as compute, storage, data transfer, monitoring, and logging.
 
+Each cloud infrastructure feature is priced as an EDB BigAnimal Infrastructure Unit (BIU). The nominal value of a BIU is $1.00, so that 1 BIU equals $1. For example, an m5.xlarge instance where $0.192/hour is the instance type cost in AWS is multiplied by 1 BIU so that the total cost is $0.192/hour. 
```

Comment:
"Each cloud infrastructure feature is priced as an EDB BigAnimal Infrastructure Unit (BIU). The nominal value of a BIU is $1.00, so that 1 BIU equals $1. For example, an m5.xlarge instance where $0.192/hour is the instance type cost in AWS is multiplied by 1 BIU so that the total cost is $0.192/hour. "

Replace with:
"A BigAnimal Infrastructure Unit (BIU) is a unit of measure for cloud infrastructure resources, such as compute, storage, and data transfer. The nominal value of one BIU is $1.00. For example, an AWS compute instance priced at $0.192/instance-hour is multiplied by 1 BIU, so that the equivalent BIU cost is $0.192/BIU-hour. 

Within the BigAnimal portal, you can see a summary of the estimated charges for the database usage and BigAnimal infrastructure units.